e2e/e2e-example.sh

```diff
@@ -3,6 +3,7 @@
 # SPDX-License-Identifier: Apache-2.0
 ENVOY_HOST=${ENVOY_HOST:-"localhost:8080"}
 HTTPBIN_HOST=${HTTPBIN_HOST:-"localhost:8081"}
+TIMEOUT_SECS=${TIMEOUT_SECS:-5}
```

Comment:
Envoy 1.24 is currently not working as the request gets trapped. This timeout allows to timeout the request in such case.